tsconfig.json

```diff
@@ -22,10 +22,7 @@
             ],
             "ngx-schema-form": [
                 "develop/ngx-schema-form/projects/schema-form/src/lib"
-            ],
-            "pastanaga": [
```

Comment:
we do need this checkout here

Reply:
@bloodbare no no, I'll do it, but I planned to merge once the whole thing is working (forms are now displayed properly in executioner, I made all the schema fixes for that, but I need to implement an custom array widget in nsf-pastanaga because the default one is not ok).

I should be able to do that next weekend.

By the way, did you noticed my issue: https://github.com/plone/guillotina_cms/issues/18 ?
We will need a solution if we want the forms to work (but if you think it is better to call vocabularies endpoints that's also fine with me).